### PR TITLE
Setting seed for random in addition to numpy and torch

### PR DESCRIPTION
**Pull request by arnaghosh**
**Summary of changes**
1. Updated `set_seeds` function to set the seed for `random` and tried using torch deterministic algorithms for reproducibility: Note that this did not improve reproducibility of numbers because ffcv with multiple workers is the primary reason for variability. In future experiments, we should run SSL training for 3 seeds and simultaneously train linear eval for each SSL model to average over variability.
2. Added `sbatch_sh_files/verify_barlowtwins.sh`: Added a simple script file to run BarlowTwins training for ResNet-50 backbone with the best hyperparameters for Cifar-10. This should enable easy experiments and testing in the future.

### CODE REVIEW

File: sbatch_sh_files/verify_barlowTwins.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+dataset='cifar10'
+if [ $dataset = 'stl10' ]
+then
+    batch_size=256
+else
+    batch_size=512
+fi
+
+checkpt_dir=$SCRATCH/fastssl/checkpoints
+
```
Comment (kumarkrishna) [inline sbatch_sh_files/verify_barlowTwins.sh:28]: can we add a line for each script lines below for readability/documentation?

**Review by kumarkrishna — APPROVED** [review]
lgtm

File: sbatch_sh_files/verify_barlowTwins.sh (hunk 1 above)
Comment (arnaghosh) [reply sbatch_sh_files/verify_barlowTwins.sh:28]: Yes, I have done some of that in the other PR. I will add more details to improve readability.
